packages/components/src/components/form/FractionButtons/FractionButtons.tsx

```diff
@@ -0,0 +1,43 @@
+import { spacings } from '@trezor/theme';
+import { Button, Row, Tooltip } from '@trezor/components';
+
+import { ButtonVariant } from '../../buttons/buttonStyleUtils';
+
+const DEFAULT_VARIANT = 'tertiary';
+
+export type FractionButton = {
+    label: string;
+    translation?: React.ReactNode;
+    tooltip: React.ReactNode;
+    isDisabled?: boolean;
+    variant?: ButtonVariant;
+    onClick: () => void;
+};
+
+export type FractionButtonsProps = {
+    buttons: FractionButton[];
+};
+
```

Comment:
We usually don't use this pattern, but rather subcomponents (like FractionButtons.Button). But in this case, I'd just export FractionButton, the container doesn't do anything special.